chapter06/mdw/Ex01.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+
```

Comment:
I'm gonna have to be strict and warn you again about not using strict and warnings